sdk/bootstrap/main.go

```diff
@@ -29,6 +29,11 @@ const (
 
 var verbose = false
 
+// The following replaces the keys with the values when logging.
+var blackListedEnvironKeys = map[string]string{
```

Comment:
It would be great if we could have a test for this so as to prevent regressions in the future.